src/values/mod.rs

```diff
@@ -195,7 +202,7 @@ impl<'ctx> Value<'ctx> {
     /// Sets the section of the global value
     fn set_section(self, section: Option<&str>) {
         #[cfg(target_os = "macos")]
-        let section = section.map(|s| format!(",{}", s));
+        let section = section.map(|s| if s.contains(",") { format!("{}", s) } else { format!(",{}", s) });
```

Comment:
Should this be starts_with?

Reply:
No. the idea is if a section name is just "mystuff", it needs to be set as ",mystuff", but if it's "XXX,mystuff", then don't touch it.
Section are expected to be in the format "<segment name>,<section name>", and it seems a no segment-name is acceptable, that why the previous commit added the ",".